big_tests/tests/mongoose_helper.erl

```diff
@@ -210,9 +212,13 @@ forget_persistent_rooms() ->
     rpc(mim(), mnesia, clear_table, [muc_registered]),
     ok.
 
--spec successful_rpc(atom(), atom(), list()) -> term().
+-spec successful_rpc(M :: atom(), F :: atom(), A :: list()) -> term().
```

Comment:
Does `MFA` even need typespecs? :D

Reply:
There are even [builtin `mfa/0` and `module/0` types](http://erlang.org/doc/reference_manual/typespec.html#types-and-their-syntax).